invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -123,6 +125,7 @@ def user_dashboard_request_view(request, **kwargs):
             is_user_dashboard=True,
             custom_fields_ui=load_custom_fields()["ui"],
             user_communities_memberships=get_user_communities_memberships(),
+            external_resources=external_resources(record),
```

Comment:
minor: oof, is this needed for this specific template? I know there's another template for previewing a record, but I'm not sure where exactly it is... I would ask someone familiar with deposit form/UI to get the right place.

Reply:
it is needed, I asked to include it there, otherwise the request's record tab is not displayed properly with full info